### ISSUE TITLE
auto format stops working when using the following sintax

### ISSUE THREAD

**Issue by mpmont** [driade/phpfmt8#12]
Using typed functions where I want a fallback return or the params will have more than one type it does not work. 

Example:

```
protected function allData(object|null $proposal): object|null
{
    return $proposal;
}
```

Basically it says that I have a syntax error and it does not run the auto format in this file.

My current local PHP version being used by FMT is the following:

```
php -v
PHP 8.2.2 (cli) (built: Feb  1 2023 08:33:04) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.2, Copyright (c) Zend Technologies
```

**Comment by driade** [comment]
Hi. I can't reproduce the issue. The message you're obtaining seems to comes from another plugin or place, if phpfmt hasn't changed your code (I've tried your sample with php 8.2 and it doesn't).

I cannot find the option "check php file on save" in phpfmt. I've it activated too somewhere in my own SublimeText...

**Comment by driade** [comment]
Please be aware that the phpbin option at the phpfmt config should be 8.2 in this case

**Comment by mpmont** [comment]
Yeah, let's close this one. I still had my config pointing to the php 7 that was used by the old plugin.
